integration/shared/isolated/marketplace_command_test.go

```diff
@@ -171,9 +171,9 @@ var _ = Describe("marketplace command", func() {
 						})
 					})
 
-					When("cc api version >= 2.125.0", func() {
+					When("CC API returns broker names in response", func() {
 						BeforeEach(func() {
-							helpers.SkipIfVersionLessThan(ccversion.MinVersionServiceBrokerNameV2)
+							helpers.SkipIfVersionAtLeast(ccversion.MinVersionMultiServiceRegistrationV2)
```

Comment:
This test currently fails in our legacy environment. I am not sure if this change from `SkipIfVersionLessThan` to `SkipIfVersionAtLeast` was intentional or not. The reason for the test failure is that for version below `2.125.0`, the api does not support broker name (This field `broker.Name` is empty) but the current implementation relies on this being available.

Reply:
This change was just brain not working correctly when thinking about the `When` and the `Skip`. I thought it would read correctly this way but actually flipped the conditions. I've changed the conditions back and I think it should be good now?